tests/plugins/emoji/basic.js

```diff
@@ -183,6 +183,19 @@
 				assert.areSame( 1, autocomplete.model.data.length, 'Emoji result contains more than one result' );
 				objectAssert.areEqual( { id: ':collision:', symbol: '💥' }, { id: autocomplete.model.data[ 0 ].id, symbol: autocomplete.model.data[ 0 ].symbol }, 'Emoji result contains wrong result' );
 			} );
+		},
+
+		// (#2527)
+		'test emoji autocomplete is sorted in proper order': function( editor, bot ) {
```

Comment:
I'd actually like to check the entire data set. Here you're checking just the first value.

I'll modify this unit test as a part of review.